hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/TestPmemCacheRestore.java

```diff
@@ -293,16 +289,12 @@ public Boolean get() {
       // The cachePath shouldn't be null if the replica has been cached
       // to pmem.
       assertNotNull(cachePath);
-      String expectFileName =
-          PmemVolumeManager.getInstance().setAndGetFileName(key);
       if (cachePath.startsWith(PMEM_DIR_0)) {
-        assertTrue(cachePath.equals(PmemVolumeManager
-            .getRealPmemDir(PMEM_DIR_0) + "/" + key.getBlockPoolId() +
-            "/" + expectFileName));
+        assertTrue(cachePath.equals(PmemVolumeManager.getInstance().
```

Comment:
cachePath is obtained from #inferCacheFilePath, so it should not be tested like this.

Reply:
Essentially, cachePath is obtained by idToCacheFilePath, so it looks that such test doesn't make sense.